orchestrate-containers/collab.md

```diff
@@ -737,3 +932,148 @@ kubectl create -f deployWordPress.yml
 
 You can now access your WordPress website via your sub-domain/public IP, e.g. `http://subdomain1.commandocloudlet.com`.
 
+## Ansible scripts
```

Comment:
File long scripts in a folder in the repo. I mentioned this before hor...

Reply:
My bad, I will create a folder in the repo.